modules/ztp-install-factory-spoke-pipeline.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="run-the-edge-cluster_factory_install_pipeline_{context}"]
+= Run the edge cluster factory install pipeline
+
+The edge factory install pipeline is as follows.
```

Comment:
Should this say something like you do elsewhere, about "Run the following steps to..." ?